snapshot/cli/cli.go

```diff
@@ -119,3 +127,36 @@ func (c *ingestGitCommitCommand) run(db snapshot.DB, _ *cobra.Command, _ []strin
 	fmt.Println(id)
 	return nil
 }
+
+type exportGitCommitCommand struct {
+	id string
+}
+
+func (c *exportGitCommitCommand) register() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "export_git_commit",
```

Comment:
I feel like this and ingest have awkward naming since it's trying to be more general than the current main.go requires. Can we get something closer to the intent which is 'import from store' and 'export to store'?